packages/context-editor/src/style.css

```diff
@@ -170,3 +181,96 @@
 	height: 182px !important; /* Ensure it takes 100% of its parent container's height */
 	min-height: unset !important; /* Remove the default min-height */
 }
+
+/* Table */
+.ProseMirror table {
+	border-spacing: 0;
+	border-collapse: separate;
+	font-size: 0.9em;
+	font-family: sans-serif;
+	min-width: 400px;
+	width: 100%;
+	table-layout: fixed;
+	overflow: hidden;
+}
+
+.ProseMirror table td,
+.ProseMirror table th {
+	border-top: 1px solid #ccc;
+	border-left: 1px solid #ccc;
+}
+
+.ProseMirror table td:last-child,
+.ProseMirror table th:last-child {
+	border-right: 1px solid #ccc;
+}
+
+.ProseMirror table tr:last-child td {
+	border-bottom: 1px solid #ccc;
+}
+
+.ProseMirror table td {
+	vertical-align: top;
+}
+
+.ProseMirror table {
+	border-left: 1px solid #777;
+}
+
+.wrap-widget + table {
+	position: relative;
+	top: -1px;
+}
+
+.wrap-widget button {
+	z-index: 10;
+}
```

Comment:
why is this necessary? it is causing the widgets to overlap with the menu

![image](https://github.com/user-attachments/assets/d45a35ae-7362-489d-83da-698b0d0f938d)

id suggest removing it


```suggestion

```